lib/promiscuous_black_hole/operation.rb

```diff
@@ -5,8 +5,9 @@
 
 module Promiscuous::BlackHole
   class Operation
-    def initialize(message)
+    def initialize(message, embedded: false)
```

Comment:
if there are many other changes, i'd want to break this out, but i think the flag is a fine solution for now